framework/base/Controller.php

```diff
@@ -479,40 +479,10 @@ public function setViewPath($path)
      * Please refer to [[render()]] on how to specify this parameter.
      * @throws InvalidParamException if an invalid path alias is used to specify the layout.
      */
-    public function findLayoutFile($view)
+    public function findLayoutFile(View $view)
```

Comment:
Method signature can't be changed in Yii 2.0.x because of backwards compatibility.